torch/_inductor/codecache.py

```diff
@@ -1812,7 +1812,42 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
                 for entry in gpu_codecache.cache.values()
                 if entry.output_path.endswith(".o")
             ]
-            gpu_kernels_o = " ".join(gpu_kernels_o)
+
+            cubins_o = []
+            if config.aot_inductor.embed_cubin:
+                # Embed cubin files into .so using objcopy
+                if config.is_fbcode():
+                    ld = build_paths.ld
+                    objcopy = (
+                        build_paths.objcopy_fallback
+                        if use_relative_path
+                        else build_paths.objcopy
+                    )
+                else:
+                    ld = "ld"
+                    objcopy = "objcopy"
+
+                for kernel_name, value in CudaKernelParamCache.cache.items():
+                    cubin_file = value[get_cpp_wrapper_cubin_path_name()]
+                    obj_file = cubin_file + ".o"
+                    cubins_o.append(obj_file)
+                    # Convert .cubin to .o
+                    cmd = f"{ld} -r -b binary -z noexecstack -o {obj_file} {cubin_file}"
```

Comment:
Would this syntax work for llvm-ld? Or is this a non-issue?

Reply:
I tried `lld` locally and it worked. Is there any reason you want to use `llvm-ld`?

Reply:
Just worried that on some systems `ld` might be symlinked to `llvm-ld`, and then this might bite. But maybe it's a non-issue

Reply:
E.g. maybe on Apple machines where clang/llvm toolchain is the default...